CHANGELOG.md

```diff
@@ -47,6 +47,18 @@ suffix.
 
 - Add configuration option for `migration_path` [#4190](https://github.com/solidusio/solidus/pull/4190) ([SuperGoodSoft](https://github.com/supergoodsoft/))
 
+### Frontend
+
+DEPRECATION WARNING: SolidusFrontend is deprecated. It will be removed from the
+Solidus meta-package gem in a future version of Solidus. Furthermore, its code
```

Comment:
I'd say "solidus meta-package" (with initial "s" in lower case), as that's the name of the gem, and people will be less confused.

Maybe we can specify that the Solidus version where it'll be removed will be 4.0.

Other than that, it's very clear :+1:

Reply:
> I'd say "solidus meta-package" (with initial "s" in lower case), as that's the name of the gem, and people will be less confused.

I'm okay with that, since "solidus" specifically refers to the gem. But how about SolidusFrontend and Solidus in "in a future version of Solidus"? For those, let's keep the camel case?